streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -1291,7 +1291,6 @@ private static <K, V> List<ConsumerRecord<K, V>> readRecords(final String topic,
                                                                  final int maxMessages) {
         final List<ConsumerRecord<K, V>> consumerRecords;
         consumer.subscribe(singletonList(topic));
-        System.out.println("Got assignment:" + consumer.assignment());
```

Comment:
This was a stray sout from one of my earlier PRs which landed up in trunk. Didn't want to create a separate PR just for this :D